product-mini/platforms/linux-sgx/enclave-sample/App/App.cpp

```diff
@@ -97,11 +97,12 @@ enclave_init(sgx_enclave_id_t *p_eid)
      */
     /* try to get the token saved in $HOME */
     home_dir = getpwuid(getuid())->pw_dir;
+    size_t home_dir_len = strlen(home_dir);
```

Comment:
Should check home_dir firstly to avoid strlen(NULL), e.g.:
```C
size_t home_dir_len = home_dir ? strlen(home_dir) : 0;
```

Reply:
Fixed,thanks for your comments. @wenyongh